packages/app/src/components/SearchPage/SearchResult.jsx

```diff
@@ -285,6 +285,15 @@ class SearchResult extends React.Component {
 
     const listView = this.renderListView(this.props.pages);
 
+    let totalValue;
+
+    if (typeof this.props.searchResultMeta.total === 'string') {
+      totalValue = this.props.searchResultMeta.total; // elasticsearch V6
+    }
+    else {
+      totalValue = this.props.searchResultMeta.total.value; // elasticsearch v7
+    }
```

Comment:
This difference between v6/v7 should be handled by the server.

Please fix in https://youtrack.weseek.co.jp/issue/GW-7608.